code/_globalvars/bitfields.dm

```diff
@@ -440,6 +440,9 @@ GLOBAL_LIST_INIT(bitfields, list(
 		"CHAT_STATISTICS" = CHAT_STATISTICS,
 		"CHAT_LOOC" = CHAT_LOOC
 		),
+	"toggles_alerts" = list(
+		"ALERTS_XENO_HEALTH" = ALERTS_XENO_HEALTH
+		),
```

Comment:
Toggles chat exists for a reason

Reply:
This is more about tactical information, but sure